QUALITY_DECLARATION.md

```diff
@@ -0,0 +1,91 @@
+
+	/*********************************************************************
+	* Software License Agreement (BSD License)
+	* *
+	* * Copyright (c) 2008, Willow Garage, Inc.
+	* * All rights reserved.
+	* *
+	* * Redistribution and use in source and binary forms, with or without
+	* * modification, are permitted provided that the following conditions
+	* * are met:
+	*
+	* * Redistributions of source code must retain the above copyright
+	* notice, this list of conditions and the following disclaimer.
+	* * Redistributions in binary form must reproduce the above
+	* copyright notice, this list of conditions and the following
+	* disclaimer in the documentation and/or other materials provided
+	* with the distribution.
+	* * Neither the name of the Willow Garage nor the names of its
+	* contributors may be used to endorse or promote products derived
+	* from this software without specific prior written permission.
+	*
+	* THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+	* "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+	* LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+	* FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+	* COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+	* INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+	* BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+	* LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+	* CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+	* LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+	* ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+	* POSSIBILITY OF SUCH DAMAGE.
+	*********************************************************************/
+
+## Testing
+
+The `libconsole-bridge-dev` library includes a simple testing to check if it compiles properly the basic simple case use. Does not include any kind of performance tests, coverage information nor static analysis.
+
+## Dependencies
+
+There is no dependency on external sources for the `libconsole-bridge-dev` library, only C++ libraries included in the standard library (mutex, iostream).
+
+## Platform Support
+
+There is no particular statement regarding this library provides, or not, support for any particular platform. The fact that it does not use external dependencies to C++ makes it portable enough to consider it compatible with Tier1 ROS2 platforms. But there is no public information related from this available on its repository.
+
+## Conclusion summary
+
+The `libconsole-bridge-dev` has a simple case test and license headers over its source files. However, does not cover several of the standards defined for quality level 1 packages. It was decided by the XYZ committee that the `libconsole-bridge-dev` library will not be considered quality level 1.
```

Comment:
Two things to consider: 
1. Implicit declaration of a team that will take the decision of considering external libraries level 1 or not (if needed be, for particular use cases)
2. If external libraries are downloaded at a fixed version, maybe external dependencies can be evaluated directly by making internal test cases with the subset of the functionality of the library that's used in the ROS2 packages ?